Libs/MRML/Widgets/qMRMLVolumeInfoWidget.cxx

```diff
@@ -186,6 +186,15 @@ void qMRMLVolumeInfoWidget::updateWidgetFromMRML()
     double origin[3] = {0.,0.,0.};
     d->ImageOriginWidget->setCoordinates(origin);
 
+    double IJKToRASDirections[3][3] = { {1,0,0}, {0,1,0}, {0,0,1} };
```

Comment:
Double values should be initialized with double (`1.0` or just `1.`). But I would rather remove this line and just use `i==j ? 1.0 : 0.0`). Also add a comment that we are setting the matrix to identity.